tests/common/experimental/test_reducers_and_aggregators.py

```diff
@@ -235,6 +235,54 @@ def test_quantile_reducers(self, reducer_name, ref, reducers):
         for i, ref_ in enumerate(ref):
             assert fns.allclose(val[i], self.get_nncf_tensor(ref_))
 
+    @pytest.mark.parametrize(
+        "axes, np_data, reference",
+        [
+            [None, np.array([[[1, 2, 0], [1, -3, 10]], [[-1, 2, -3], [4, 5, -6]]]), 16.1666],
+            [(0,), np.array([[[1, 2, 0], [1, -3, 10]], [[-1, 2, -3], [4, 5, -6]]]), 14.25],
+            [(0, 1), np.array([[[1, 2, 0], [1, -3, 10]], [[-1, 2, -3], [4, 5, -6]]]), 15.875],
```

Comment:
As each ref input is equal to each other, I believe we can create a global var like `DEFAULT_VARIANCE_REF_INPUT` 😀